src/extension.ts

```diff
@@ -15,7 +16,8 @@ import {
 import makeGraph from "./makeGraph";
 
 export async function activate(context: vscode.ExtensionContext) {
-  const decorations = new Decorations();
+  const fontSize = await computeFontSize();
```

Comment:
yeah I guess prob better to hoist all that stuff out of the constructor 😂

Reply:
Hah yeah.  async constructor is problematic also ;)